Core/include/Acts/Propagator/EigenStepper.ipp

```diff
@@ -268,6 +268,10 @@ Acts::Result<double> Acts::EigenStepper<E, A>::step(
                          state.options.tolerance / std::abs(error_estimate))))),
             4.0f);
   }
+
+  // TODO: enable only if needed
```

Comment:
I think you can always do that, this operation doesn't cost any time and checking would be an overkill for this.